src/software/amazon/ion/Timestamp.java

```diff
@@ -1252,6 +1121,13 @@ public static Timestamp forMillis(long millis, Integer localOffset)
      * Returns a Timestamp that represents the point in time that is
      * {@code millis} milliseconds (including any fractional
      * milliseconds) from the epoch, with a given local offset.
+     * A default {@link GregorianCalendar} will be used to perform any
+     * arithmetic operations on the resulting Timestamp. NOTE: this means
```

Comment:
suggestion: use html tags to highlight the `NOTE`, e.g.
```
... Timestamp.

<p>
<strong> NOTE: </strong> this means ....
</p>
```

Reply:
Done.